plugin-modernizer-core/src/test/java/io/jenkins/tools/pluginmodernizer/core/utils/PluginServiceTest.java

```diff
@@ -151,6 +151,30 @@ public void shouldExtractRepoNameForLocalDefaultPluginWithGitHubRepo() throws Ex
         assertEquals("foobar", result);
     }
 
+    @Test
+    public void shouldExtractRepoNameForLocalDefaultPluginWithScmRepoRepo() throws Exception {
+        PluginService service = getService();
+
+        // language=xml
+        String pom =
+                """
+                    <project>
+                        <properties>
+                            <scm>
+                               <connection>scm:git:https://github.com/jenkinsci/FOO_Bar-plugin.git</connection>
```

Comment:
For example https://github.com/jenkinsci/JDK_Parameter_Plugin-plugin look like this